<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font" >
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle" >
   <string>统计</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="pygrid.qrc" >
    <normaloff>:/icon/stat.png</normaloff>:/icon/stat.png</iconset>
  </property>
  <property name="locale" >
   <locale country="China" language="Chinese" />
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" name="verticalLayout" >
    <item>
     <widget class="QGroupBox" name="groupBox" >
      <property name="title" >
       <string>设置筛选条件</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <item>
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>日期：</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
         <property name="buddy" >
          <cstring>dateEdit_From</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateEdit_From" >
         <property name="displayFormat" >
          <string>yyyy-MM-dd</string>
         </property>
         <property name="calendarPopup" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>至</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
         <property name="buddy" >
          <cstring>dateEdit_To</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateEdit_To" >
         <property name="displayFormat" >
          <string>yyyy-MM-dd</string>
         </property>
         <property name="calendarPopup" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>品名：</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
         <property name="buddy" >
          <cstring>comboBox_SearchName</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_SearchName" >
         <property name="editable" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>类别：</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
         <property name="buddy" >
          <cstring>comboBox_SearchCategory</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_SearchCategory" >
         <property name="editable" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Stat" >
         <property name="text" >
          <string>统计</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget" >
      <property name="currentIndex" >
       <number>4</number>
      </property>
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>总体统计</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2" >
        <item>
         <widget class="QTableWidget" name="tableWidget_OverView" >
          <property name="editTriggers" >
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <column>
           <property name="text" >
            <string>支出</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>收入</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>余额</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2" >
       <attribute name="title" >
        <string>按品名分组</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3" >
        <item>
         <widget class="QTableWidget" name="tableWidget_NameGroup" >
          <property name="editTriggers" >
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <column>
           <property name="text" >
            <string>品名</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>支出</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>收入</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>余额</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3" >
       <attribute name="title" >
        <string>按类别分组</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4" >
        <item>
         <widget class="QTableWidget" name="tableWidget_CategoryGroup" >
          <property name="editTriggers" >
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <column>
           <property name="text" >
            <string>类别</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>支出</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>收入</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>余额</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4" >
       <attribute name="title" >
        <string>按月分组</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_5" >
        <item>
         <widget class="QTableWidget" name="tableWidget_MonthGroup" >
          <property name="editTriggers" >
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <column>
           <property name="text" >
            <string>月份</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>支出</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>收入</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>余额</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5" >
       <attribute name="title" >
        <string>按年分组</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_6" >
        <item>
         <widget class="QTableWidget" name="tableWidget_YearGroup" >
          <property name="editTriggers" >
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <column>
           <property name="text" >
            <string>年份</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>支出</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>收入</string>
           </property>
          </column>
          <column>
           <property name="text" >
            <string>余额</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F" >
    <property name="title" >
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action_Exit" />
   </widget>
   <widget class="QMenu" name="menu_O" >
    <property name="title" >
     <string>操作(&amp;O)</string>
    </property>
    <addaction name="action_Print" />
   </widget>
   <addaction name="menu_F" />
   <addaction name="menu_O" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle" >
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="action_Print" />
  </widget>
  <action name="action_Exit" >
   <property name="icon" >
    <iconset resource="pygrid.qrc" >
     <normaloff>:/icon/exit.png</normaloff>:/icon/exit.png</iconset>
   </property>
   <property name="text" >
    <string>退出(&amp;X)</string>
   </property>
   <property name="statusTip" >
    <string>点这个就退出了，快捷键:Ctrl+Q</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Print" >
   <property name="icon" >
    <iconset resource="pygrid.qrc" >
     <normaloff>:/icon/print.png</normaloff>:/icon/print.png</iconset>
   </property>
   <property name="text" >
    <string>打印(&amp;P)</string>
   </property>
   <property name="statusTip" >
    <string>把统计的结果打印出来，快捷键:Ctrl+P</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+P</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="pygrid.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>action_Exit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
